### ISSUE TITLE
Simplify python dependency

### ISSUE THREAD

**Issue by lgarithm** [tensorlayer/openpose-plus#115]
In order to export a model defined in TL to uff format, we need the following pip packages

- tensorlayer: installed via `pip`
- uff: installed via `dpkg` from a local deb package provided by nvidia

However `uff` only supports python 3.5, but the default python3 on ubuntu18 is python3.6.
If we install TL via `python3.5 -m pip install tensorlayer --user`, there will be some missing pip packages, which must be installed via `apt` instead of `pip`, i.e. `tk`.

**Comment by lgarithm** [comment]
Currently the local repo `nv-tensorrt-repo-ubuntu1604-cuda9.0-ga-trt4.0.1.6-20180612_1-1_amd64.deb` contains:

```
$ dpkg -c nv-tensorrt-repo-ubuntu1604-cuda9.0-ga-trt4.0.1.6-20180612_1-1_amd64.deb 
drwxr-xr-x root/root         0 2018-06-13 13:48 ./
drwxr-xr-x root/root         0 2018-06-13 13:48 ./var/
drwxr-xr-x root/root         0 2018-06-13 13:48 ./var/nv-tensorrt-repo-cuda9.0-ga-trt4.0.1.6-20180612/
-rw-r--r-- root/root  37392964 2018-06-13 13:48 ./var/nv-tensorrt-repo-cuda9.0-ga-trt4.0.1.6-20180612/libnvinfer-dev_4.1.2-1+cuda9.0_amd64.deb
-rw-r--r-- root/root      1122 2018-06-13 13:48 ./var/nv-tensorrt-repo-cuda9.0-ga-trt4.0.1.6-20180612/python3-libnvinfer-dev_4.1.2-1+cuda9.0_amd64.deb
-rw-r--r-- root/root       574 2018-06-13 13:48 ./var/nv-tensorrt-repo-cuda9.0-ga-trt4.0.1.6-20180612/Release
-rw-r--r-- root/root   2061302 2018-06-13 13:48 ./var/nv-tensorrt-repo-cuda9.0-ga-trt4.0.1.6-20180612/python3-libnvinfer-doc_4.1.2-1+cuda9.0_amd64.deb
-rw-r--r-- root/root   2061530 2018-06-13 13:48 ./var/nv-tensorrt-repo-cuda9.0-ga-trt4.0.1.6-20180612/python-libnvinfer-doc_4.1.2-1+cuda9.0_amd64.deb
-rw-r--r-- root/root     11860 2018-06-13 13:48 ./var/nv-tensorrt-repo-cuda9.0-ga-trt4.0.1.6-20180612/graphsurgeon-tf_4.1.2-1+cuda9.0_amd64.deb
-rw-r--r-- root/root     33528 2018-06-13 13:48 ./var/nv-tensorrt-repo-cuda9.0-ga-trt4.0.1.6-20180612/uff-converter-tf_4.1.2-1+cuda9.0_amd64.deb
-rw-r--r-- root/root      1120 2018-06-13 13:48 ./var/nv-tensorrt-repo-cuda9.0-ga-trt4.0.1.6-20180612/libgie-dev_4.1.2-1+cuda9.0_amd64.deb
-rw-r--r-- root/root   1508550 2018-06-13 13:48 ./var/nv-tensorrt-repo-cuda9.0-ga-trt4.0.1.6-20180612/tensorrt_4.0.1.6-1+cuda9.0_amd64.deb
-rw-r--r-- root/root 114862234 2018-06-13 13:48 ./var/nv-tensorrt-repo-cuda9.0-ga-trt4.0.1.6-20180612/libcudnn7-dev_7.1.3.16-1+cuda9.0_amd64.deb
-rw-r--r-- root/root      1122 2018-06-13 13:48 ./var/nv-tensorrt-repo-cuda9.0-ga-trt4.0.1.6-20180612/python-libnvinfer-dev_4.1.2-1+cuda9.0_amd64.deb
-rw-r--r-- root/root 124664466 2018-06-13 13:48 ./var/nv-tensorrt-repo-cuda9.0-ga-trt4.0.1.6-20180612/libcudnn7_7.1.3.16-1+cuda9.0_amd64.deb
-rw-r--r-- root/root       819 2018-06-13 13:48 ./var/nv-tensorrt-repo-cuda9.0-ga-trt4.0.1.6-20180612/Release.gpg
-rw-r--r-- root/root  36070364 2018-06-13 13:48 ./var/nv-tensorrt-repo-cuda9.0-ga-trt4.0.1.6-20180612/libnvinfer4_4.1.2-1+cuda9.0_amd64.deb
-rw-r--r-- root/root   1035870 2018-06-13 13:48 ./var/nv-tensorrt-repo-cuda9.0-ga-trt4.0.1.6-20180612/python-libnvinfer_4.1.2-1+cuda9.0_amd64.deb
-rw-r--r-- root/root      1633 2018-06-13 13:48 ./var/nv-tensorrt-repo-cuda9.0-ga-trt4.0.1.6-20180612/7fa2af80.pub
-rw-r--r-- root/root   1044464 2018-06-13 13:48 ./var/nv-tensorrt-repo-cuda9.0-ga-trt4.0.1.6-20180612/python3-libnvinfer_4.1.2-1+cuda9.0_amd64.deb
-rw-r--r-- root/root      3618 2018-06-13 13:48 ./var/nv-tensorrt-repo-cuda9.0-ga-trt4.0.1.6-20180612/Packages.gz
-rw-r--r-- root/root 270923918 2018-06-13 13:48 ./var/nv-tensorrt-repo-cuda9.0-ga-trt4.0.1.6-20180612/libnvinfer-samples_4.1.2-1+cuda9.0_amd64.deb
drwxr-xr-x root/root         0 2018-06-13 13:48 ./etc/
drwxr-xr-x root/root         0 2018-06-13 13:48 ./etc/apt/
drwxr-xr-x root/root         0 2018-06-13 13:48 ./etc/apt/sources.list.d/
-rw-r--r-- root/root        66 2018-06-13 13:48 ./etc/apt/sources.list.d/nv-tensorrt-cuda9.0-ga-trt4.0.1.6-20180612.list
drwxr-xr-x root/root         0 2018-06-13 13:48 ./usr/
drwxr-xr-x root/root         0 2018-06-13 13:48 ./usr/share/
drwxr-xr-x root/root         0 2018-06-13 13:48 ./usr/share/doc/
drwxr-xr-x root/root         0 2018-06-13 13:48 ./usr/share/doc/nv-tensorrt-repo-ubuntu1604-cuda9.0-ga-trt4.0.1.6-20180612/
-rw-r--r-- root/root       148 2018-06-13 13:48 ./usr/share/doc/nv-tensorrt-repo-ubuntu1604-cuda9.0-ga-trt4.0.1.6-20180612/changelog.Debian.gz
```

and `/var/nv-tensorrt-repo-cuda9.0-ga-trt4.0.1.6-20180612/uff-converter-tf_4.1.2-1+cuda9.0_amd64.deb` contains:

```
$ dpkg -c /var/nv-tensorrt-repo-cuda9.0-ga-trt4.0.1.6-20180612/uff-converter-tf_4.1.2-1+cuda9.0_amd64.deb 
drwxr-xr-x root/root         0 2018-06-12 17:45 ./
drwxr-xr-x root/root         0 2018-06-12 17:44 ./usr/
drwxr-xr-x root/root         0 2018-06-12 17:44 ./usr/lib/
drwxr-xr-x root/root         0 2018-06-12 17:44 ./usr/lib/python3.5/
drwxr-xr-x root/root         0 2018-06-12 17:44 ./usr/lib/python3.5/dist-packages/
drwxr-xr-x root/root         0 2018-06-12 17:44 ./usr/lib/python3.5/dist-packages/uff-0.4.0.dist-info/
-rw-r--r-- root/root        64 2018-06-12 17:44 ./usr/lib/python3.5/dist-packages/uff-0.4.0.dist-info/entry_points.txt
-rw-r--r-- root/root       110 2018-06-12 17:44 ./usr/lib/python3.5/dist-packages/uff-0.4.0.dist-info/WHEEL
-rw-r--r-- root/root      4253 2018-06-12 17:44 ./usr/lib/python3.5/dist-packages/uff-0.4.0.dist-info/RECORD
-rw-r--r-- root/root      1458 2018-06-12 17:44 ./usr/lib/python3.5/dist-packages/uff-0.4.0.dist-info/METADATA
-rw-r--r-- root/root         1 2018-06-12 17:44 ./usr/lib/python3.5/dist-packages/uff-0.4.0.dist-info/zip-safe
-rw-r--r-- root/root         4 2018-06-12 17:44 ./usr/lib/python3.5/dist-packages/uff-0.4.0.dist-info/top_level.txt
drwxr-xr-x root/root         0 2018-06-12 17:44 ./usr/lib/python3.5/dist-packages/uff/
-rw-r--r-- root/root       442 2018-06-12 17:44 ./usr/lib/python3.5/dist-packages/uff/__init__.py
drwxr-xr-x root/root         0 2018-06-12 17:44 ./usr/lib/python3.5/dist-packages/uff/converters/
-rw-r--r-- root/root         0 2018-06-12 17:44 ./usr/lib/python3.5/dist-packages/uff/converters/__init__.py
-rw-r--r-- root/root       999 2018-06-12 17:44 ./usr/lib/python3.5/dist-packages/uff/converters/converter_base.py
drwxr-xr-x root/root         0 2018-06-12 17:44 ./usr/lib/python3.5/dist-packages/uff/converters/tensorflow/
-rw-r--r-- root/root         0 2018-06-12 17:44 ./usr/lib/python3.5/dist-packages/uff/converters/tensorflow/__init__.py
-rw-r--r-- root/root      9208 2018-06-12 17:44 ./usr/lib/python3.5/dist-packages/uff/converters/tensorflow/converter.py
-rw-r--r-- root/root      5456 2018-06-12 17:44 ./usr/lib/python3.5/dist-packages/uff/converters/tensorflow/conversion_helpers.py
-rw-r--r-- root/root     18297 2018-06-12 17:44 ./usr/lib/python3.5/dist-packages/uff/converters/tensorflow/converter_functions.py
drwxr-xr-x root/root         0 2018-06-12 17:44 ./usr/lib/python3.5/dist-packages/uff/converters/tensorflow/proto/
-rw-r--r-- root/root       334 2018-06-12 17:44 ./usr/lib/python3.5/dist-packages/uff/converters/tensorflow/proto/__init__.py
drwxr-xr-x root/root         0 2018-06-12 17:44 ./usr/lib/python3.5/dist-packages/uff/converters/tensorflow/proto/_tensorflow_core/
-rw-r--r-- root/root         0 2018-06-12 17:44 ./usr/lib/python3.5/dist-packages/uff/converters/tensorflow/proto/_tensorflow_core/__init__.py
-rw-r--r-- root/root      5910 2018-06-12 17:44 ./usr/lib/python3.5/dist-packages/uff/converters/tensorflow/proto/_tensorflow_core/kernel_def_pb2.py
-rw-r--r-- root/root      6360 2018-06-12 17:44 ./usr/lib/python3.5/dist-packages/uff/converters/tensorflow/proto/_tensorflow_core/variable_pb2.py
-rw-r--r-- root/root      3886 2018-06-12 17:44 ./usr/lib/python3.5/dist-packages/uff/converters/tensorflow/proto/_tensorflow_core/resource_handle_pb2.py
-rw-r--r-- root/root      4480 2018-06-12 17:44 ./usr/lib/python3.5/dist-packages/uff/converters/tensorflow/proto/_tensorflow_core/allocation_description_pb2.py
-rw-r--r-- root/root      4356 2018-06-12 17:44 ./usr/lib/python3.5/dist-packages/uff/converters/tensorflow/proto/_tensorflow_core/tensor_shape_pb2.py
-rw-r--r-- root/root      5861 2018-06-12 17:44 ./usr/lib/python3.5/dist-packages/uff/converters/tensorflow/proto/_tensorflow_core/node_def_pb2.py
-rw-r--r-- root/root     16839 2018-06-12 17:44 ./usr/lib/python3.5/dist-packages/uff/converters/tensorflow/proto/_tensorflow_core/op_def_pb2.py
-rw-r--r-- root/root     10392 2018-06-12 17:44 ./usr/lib/python3.5/dist-packages/uff/converters/tensorflow/proto/_tensorflow_core/tensor_pb2.py
-rw-r--r-- root/root      8581 2018-06-12 17:44 ./usr/lib/python3.5/dist-packages/uff/converters/tensorflow/proto/_tensorflow_core/types_pb2.py
-rw-r--r-- root/root      4456 2018-06-12 17:44 ./usr/lib/python3.5/dist-packages/uff/converters/tensorflow/proto/_tensorflow_core/tensor_slice_pb2.py
-rw-r--r-- root/root      2977 2018-06-12 17:44 ./usr/lib/python3.5/dist-packages/uff/converters/tensorflow/proto/_tensorflow_core/versions_pb2.py
-rw-r--r-- root/root     18405 2018-06-12 17:44 ./usr/lib/python3.5/dist-packages/uff/converters/tensorflow/proto/_tensorflow_core/summary_pb2.py
-rw-r--r-- root/root     17528 2018-06-12 17:44 ./usr/lib/python3.5/dist-packages/uff/converters/tensorflow/proto/_tensorflow_core/attr_value_pb2.py
-rw-r--r-- root/root      4205 2018-06-12 17:44 ./usr/lib/python3.5/dist-packages/uff/converters/tensorflow/proto/_tensorflow_core/tensor_description_pb2.py
-rw-r--r-- root/root     11495 2018-06-12 17:44 ./usr/lib/python3.5/dist-packages/uff/converters/tensorflow/proto/_tensorflow_core/function_pb2.py
-rw-r--r-- root/root      4466 2018-06-12 17:44 ./usr/lib/python3.5/dist-packages/uff/converters/tensorflow/proto/_tensorflow_core/graph_pb2.py
drwxr-xr-x root/root         0 2018-06-12 17:44 ./usr/lib/python3.5/dist-packages/uff/bin/
-rw-r--r-- root/root        43 2018-06-12 17:44 ./usr/lib/python3.5/dist-packages/uff/bin/__init__.py
-rw-r--r-- root/root      3345 2018-06-12 17:44 ./usr/lib/python3.5/dist-packages/uff/bin/convert_to_uff.py
drwxr-xr-x root/root         0 2018-06-12 17:44 ./usr/lib/python3.5/dist-packages/uff/model/
-rw-r--r-- root/root       194 2018-06-12 17:44 ./usr/lib/python3.5/dist-packages/uff/model/__init__.py
-rw-r--r-- root/root     41648 2018-06-12 17:44 ./usr/lib/python3.5/dist-packages/uff/model/uff_pb2.py
-rw-r--r-- root/root      1556 2018-06-12 17:44 ./usr/lib/python3.5/dist-packages/uff/model/node.py
-rw-r--r-- root/root       145 2018-06-12 17:44 ./usr/lib/python3.5/dist-packages/uff/model/exceptions.py
-rw-r--r-- root/root      4986 2018-06-12 17:44 ./usr/lib/python3.5/dist-packages/uff/model/meta_graph.py
-rw-r--r-- root/root      4307 2018-06-12 17:44 ./usr/lib/python3.5/dist-packages/uff/model/core_descriptor.py
-rw-r--r-- root/root     13415 2018-06-12 17:44 ./usr/lib/python3.5/dist-packages/uff/model/graph.py
-rw-r--r-- root/root       559 2018-06-12 17:44 ./usr/lib/python3.5/dist-packages/uff/model/utils.py
-rw-r--r-- root/root     10914 2018-06-12 17:44 ./usr/lib/python3.5/dist-packages/uff/model/descriptor.py
-rw-r--r-- root/root      3662 2018-06-12 17:44 ./usr/lib/python3.5/dist-packages/uff/model/data.py
drwxr-xr-x root/root         0 2018-06-12 17:44 ./usr/lib/python2.7/
drwxr-xr-x root/root         0 2018-06-12 17:44 ./usr/lib/python2.7/dist-packages/
drwxr-xr-x root/root         0 2018-06-12 17:44 ./usr/lib/python2.7/dist-packages/uff-0.4.0.dist-info/
-rw-r--r-- root/root        64 2018-06-12 17:44 ./usr/lib/python2.7/dist-packages/uff-0.4.0.dist-info/entry_points.txt
-rw-r--r-- root/root       110 2018-06-12 17:44 ./usr/lib/python2.7/dist-packages/uff-0.4.0.dist-info/WHEEL
-rw-r--r-- root/root      4253 2018-06-12 17:44 ./usr/lib/python2.7/dist-packages/uff-0.4.0.dist-info/RECORD
-rw-r--r-- root/root      1458 2018-06-12 17:44 ./usr/lib/python2.7/dist-packages/uff-0.4.0.dist-info/METADATA
-rw-r--r-- root/root         1 2018-06-12 17:44 ./usr/lib/python2.7/dist-packages/uff-0.4.0.dist-info/zip-safe
-rw-r--r-- root/root         4 2018-06-12 17:44 ./usr/lib/python2.7/dist-packages/uff-0.4.0.dist-info/top_level.txt
drwxr-xr-x root/root         0 2018-06-12 17:44 ./usr/lib/python2.7/dist-packages/uff/
-rw-r--r-- root/root       442 2018-06-12 17:44 ./usr/lib/python2.7/dist-packages/uff/__init__.py
drwxr-xr-x root/root         0 2018-06-12 17:44 ./usr/lib/python2.7/dist-packages/uff/converters/
-rw-r--r-- root/root         0 2018-06-12 17:44 ./usr/lib/python2.7/dist-packages/uff/converters/__init__.py
-rw-r--r-- root/root       999 2018-06-12 17:44 ./usr/lib/python2.7/dist-packages/uff/converters/converter_base.py
drwxr-xr-x root/root         0 2018-06-12 17:44 ./usr/lib/python2.7/dist-packages/uff/converters/tensorflow/
-rw-r--r-- root/root         0 2018-06-12 17:44 ./usr/lib/python2.7/dist-packages/uff/converters/tensorflow/__init__.py
-rw-r--r-- root/root      9208 2018-06-12 17:44 ./usr/lib/python2.7/dist-packages/uff/converters/tensorflow/converter.py
-rw-r--r-- root/root      5456 2018-06-12 17:44 ./usr/lib/python2.7/dist-packages/uff/converters/tensorflow/conversion_helpers.py
-rw-r--r-- root/root     18297 2018-06-12 17:44 ./usr/lib/python2.7/dist-packages/uff/converters/tensorflow/converter_functions.py
drwxr-xr-x root/root         0 2018-06-12 17:44 ./usr/lib/python2.7/dist-packages/uff/converters/tensorflow/proto/
-rw-r--r-- root/root       334 2018-06-12 17:44 ./usr/lib/python2.7/dist-packages/uff/converters/tensorflow/proto/__init__.py
drwxr-xr-x root/root         0 2018-06-12 17:44 ./usr/lib/python2.7/dist-packages/uff/converters/tensorflow/proto/_tensorflow_core/
-rw-r--r-- root/root         0 2018-06-12 17:44 ./usr/lib/python2.7/dist-packages/uff/converters/tensorflow/proto/_tensorflow_core/__init__.py
-rw-r--r-- root/root      5910 2018-06-12 17:44 ./usr/lib/python2.7/dist-packages/uff/converters/tensorflow/proto/_tensorflow_core/kernel_def_pb2.py
-rw-r--r-- root/root      6360 2018-06-12 17:44 ./usr/lib/python2.7/dist-packages/uff/converters/tensorflow/proto/_tensorflow_core/variable_pb2.py
-rw-r--r-- root/root      3886 2018-06-12 17:44 ./usr/lib/python2.7/dist-packages/uff/converters/tensorflow/proto/_tensorflow_core/resource_handle_pb2.py
-rw-r--r-- root/root      4480 2018-06-12 17:44 ./usr/lib/python2.7/dist-packages/uff/converters/tensorflow/proto/_tensorflow_core/allocation_description_pb2.py
-rw-r--r-- root/root      4356 2018-06-12 17:44 ./usr/lib/python2.7/dist-packages/uff/converters/tensorflow/proto/_tensorflow_core/tensor_shape_pb2.py
-rw-r--r-- root/root      5861 2018-06-12 17:44 ./usr/lib/python2.7/dist-packages/uff/converters/tensorflow/proto/_tensorflow_core/node_def_pb2.py
-rw-r--r-- root/root     16839 2018-06-12 17:44 ./usr/lib/python2.7/dist-packages/uff/converters/tensorflow/proto/_tensorflow_core/op_def_pb2.py
-rw-r--r-- root/root     10392 2018-06-12 17:44 ./usr/lib/python2.7/dist-packages/uff/converters/tensorflow/proto/_tensorflow_core/tensor_pb2.py
-rw-r--r-- root/root      8581 2018-06-12 17:44 ./usr/lib/python2.7/dist-packages/uff/converters/tensorflow/proto/_tensorflow_core/types_pb2.py
-rw-r--r-- root/root      4456 2018-06-12 17:44 ./usr/lib/python2.7/dist-packages/uff/converters/tensorflow/proto/_tensorflow_core/tensor_slice_pb2.py
-rw-r--r-- root/root      2977 2018-06-12 17:44 ./usr/lib/python2.7/dist-packages/uff/converters/tensorflow/proto/_tensorflow_core/versions_pb2.py
-rw-r--r-- root/root     18405 2018-06-12 17:44 ./usr/lib/python2.7/dist-packages/uff/converters/tensorflow/proto/_tensorflow_core/summary_pb2.py
-rw-r--r-- root/root     17528 2018-06-12 17:44 ./usr/lib/python2.7/dist-packages/uff/converters/tensorflow/proto/_tensorflow_core/attr_value_pb2.py
-rw-r--r-- root/root      4205 2018-06-12 17:44 ./usr/lib/python2.7/dist-packages/uff/converters/tensorflow/proto/_tensorflow_core/tensor_description_pb2.py
-rw-r--r-- root/root     11495 2018-06-12 17:44 ./usr/lib/python2.7/dist-packages/uff/converters/tensorflow/proto/_tensorflow_core/function_pb2.py
-rw-r--r-- root/root      4466 2018-06-12 17:44 ./usr/lib/python2.7/dist-packages/uff/converters/tensorflow/proto/_tensorflow_core/graph_pb2.py
drwxr-xr-x root/root         0 2018-06-12 17:44 ./usr/lib/python2.7/dist-packages/uff/bin/
-rw-r--r-- root/root        43 2018-06-12 17:44 ./usr/lib/python2.7/dist-packages/uff/bin/__init__.py
-rw-r--r-- root/root      3345 2018-06-12 17:44 ./usr/lib/python2.7/dist-packages/uff/bin/convert_to_uff.py
drwxr-xr-x root/root         0 2018-06-12 17:44 ./usr/lib/python2.7/dist-packages/uff/model/
-rw-r--r-- root/root       194 2018-06-12 17:44 ./usr/lib/python2.7/dist-packages/uff/model/__init__.py
-rw-r--r-- root/root     41648 2018-06-12 17:44 ./usr/lib/python2.7/dist-packages/uff/model/uff_pb2.py
-rw-r--r-- root/root      1556 2018-06-12 17:44 ./usr/lib/python2.7/dist-packages/uff/model/node.py
-rw-r--r-- root/root       145 2018-06-12 17:44 ./usr/lib/python2.7/dist-packages/uff/model/exceptions.py
-rw-r--r-- root/root      4986 2018-06-12 17:44 ./usr/lib/python2.7/dist-packages/uff/model/meta_graph.py
-rw-r--r-- root/root      4307 2018-06-12 17:44 ./usr/lib/python2.7/dist-packages/uff/model/core_descriptor.py
-rw-r--r-- root/root     13415 2018-06-12 17:44 ./usr/lib/python2.7/dist-packages/uff/model/graph.py
-rw-r--r-- root/root       559 2018-06-12 17:44 ./usr/lib/python2.7/dist-packages/uff/model/utils.py
-rw-r--r-- root/root     10914 2018-06-12 17:44 ./usr/lib/python2.7/dist-packages/uff/model/descriptor.py
-rw-r--r-- root/root      3662 2018-06-12 17:44 ./usr/lib/python2.7/dist-packages/uff/model/data.py
drwxr-xr-x root/root         0 2018-06-12 17:44 ./usr/share/
drwxr-xr-x root/root         0 2018-06-12 17:44 ./usr/share/doc/
drwxr-xr-x root/root         0 2018-06-12 17:45 ./usr/share/doc/uff-converter-tf/
-rw-r--r-- root/root       157 2018-06-12 17:44 ./usr/share/doc/uff-converter-tf/changelog.Debian.gz
-rw-r--r-- root/root         0 2018-06-12 17:44 ./usr/share/doc/uff-converter-tf/copyright
```

**Comment by lgarithm** [comment]
need to install `uff` and `graphsurgeon` from `whl` files

graphsurgeon-0.2.0-py2.py3-none-any.whl
uff-0.4.0-py2.py3-none-any.whl

included in `TensorRT-4.0.1.6.Ubuntu-16.04.4.x86_64-gnu.cuda-9.0.cudnn7.1.tar.gz`
